### PR TITLE
docs: create reference cheatsheet

### PR DESCRIPTION
**Pull request by Roxie-32**
The initial draft of the reference cheatsheet. @alexgarel, kindly help out with more suggestions for this cheat sheet. The [Search API](https://wiki.openfoodfacts.org/Open_Food_Facts_Search_API_Version_2) has some AND & OR queries; I'm not sure if it's valid and if it should be here.

### CODE REVIEW

**Comment by Roxie-32** [comment]
@alexgarel I’m bringing your attention here again

File: docs/reference/api-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Open Food Facts Reference CheatSheet
+
+This reference cheatsheet shows you how to send requests to the OFF API.
```
Comment (alexgarel) [inline docs/reference/api-cheatsheet.md:3]:
```suggestion
This reference cheatsheet gives you a quick reminder to send requests to the OFF API.

If you are new to API usage you might look at the [tutorial](../tutorials/using-the-OFF-API-tutorial.md).
Also reports to the [API reference documentation](../reference/api.md) for complete information.

```

File: docs/reference/api-cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Open Food Facts Reference CheatSheet
```
Comment (alexgarel) [inline docs/reference/api-cheatsheet.md:1]:
```suggestion
# Open Food Facts API - CheatSheet
```

File: docs/reference/cheatsheet.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Open Food Facts Reference CheatSheet
```
Comment (alexgarel) [inline docs/reference/cheatsheet.md:1]: Remove this file :-)

**Review by alexgarel — COMMENTED** [review]
Some comments.

Also we should add some reference to this file, in the introduction of api.yml file, and also at the end of the tutorial maybe ?

When merging, I will also credit @aleene for he is the author of the original wiki page :-)

**Comment by Roxie-32** [comment]
@alexgarel this is ready for merge, Although , I feel there may be more things in the wiki we want to quicky remind people about. Do you have any more suggestions or its good to go?

**Review by stephanegigandet — APPROVED** [review]
I think we can merge as-is, and we'll be able to create new PRs to add to it after.

**Pull request merged**